core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -118,6 +117,49 @@ describe('Overview', () => {
             expect(jazzServices_po.getFunctionType().getText()).toEqual('function');
             expect(jazzServices_po.getFunctionStatus().getText()).toEqual('creation started');
       });
+
+      it('Verify Lamda Function Page Title', () => {
+            browser.wait(EC.visibilityOf(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            jazzServices_po.getLamdaName().click();
+            expect(jazzServices_po.getOverviewStatus().getText()).toEqual('OVERVIEW');
+            browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+            browser.sleep(45000);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getRefresh()), timeOutHigh);
+            jazzServices_po.getRefresh().click();
+            browser.sleep(30000);
+      });
+
+      it('Verify Lamda Prod Name' ,  () => {
+            browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+            jazzServices_po.getProdName().click();
+            expect(jazzServices_po.getDeploymentStatus().getText()).toEqual('DEPLOYMENTS');
+            browser.wait(EC.visibilityOf(jazzServices_po.getProdHeader()), timeOutHigh);
+            browser.wait(EC.visibilityOf(jazzServices_po.getRefresh()), timeOutHigh);
+            browser.driver.switchTo().activeElement();
+            browser.sleep(15000);
```

Comment:
these browser.sleep values work at the moment on your PC with network delays that make sense from your place to work. they may need to be tweaked if run from some other server. Putting these values in the config file would help in tweaking these values at runtime. Do you agree?

Reply:
Parth,why we are giving sleep is like,In our application For eg:one scenario,after clicking the API,UI takes some time to get it build and validate .after that only we will get the Prod properly.Its not kept to adjust with my System performance.Each and every place we are giving time for the Reg verification mail,login and deployment etc.I have a discussion with Sid about the changes of this sleep,more standard way we can give explicit values,but that we can go for a separate story with the miscellaneous items.

Reply:
understood.. lets take it up as part of a separate story